README.md

```diff
@@ -16,7 +16,7 @@ Prerequisites:
 
 Open a console and run the following command to apply this template:
  ```
-sbt -Dsbt.version=0.13.15 new akka/akka-quickstart-java.g8
+sbt -Dsbt.version=1.2.8 new akka/akka-quickstart-java.g8
```

Comment:
does it work without this `sbt.version` flag?

Reply:
Yep, we can remove, it deals with the user having an old version of sbt on their path